### ISSUE TITLE
WebRTC Statistics API

### ISSUE THREAD

**Issue by SergioMasson** [microsoft/MixedReality-WebRTC#257]
Is there a way to access the WebRTC statistics API using the MixedReality.WebRTC.Native ? Are there any plants to expose this API to the C# Wrapper?

**Comment by djee-ms** [comment]
Are you looking for [`PeerConnection.GetSimpleStatsAync()`](https://microsoft.github.io/MixedReality-WebRTC/api/Microsoft.MixedReality.WebRTC.PeerConnection.html#Microsoft_MixedReality_WebRTC_PeerConnection_GetSimpleStatsAsync)?

**Comment by SergioMasson** [comment]
Sorry, I was looking only at the API available in v1.0.3. I believe GetSimpleStatesAsync has the information I was looking for. Thank you very much!

**Comment by djee-ms** [comment]
Yes this was added on `master` only, will be available in the upcoming v2.0.

**Comment by SergioMasson** [comment]
Is there a branch where I can pull from and only get the statistics API ? Our development is stable using v1.0.3 and it is been really hard to get it to work using the master branch. We really need access to the statistics API tough.
